torch2trt/converters/view.py

```diff
@@ -45,13 +49,13 @@ def forward(self, x):
         return x.unsqueeze(dim=self.dim)        
 
 
-@add_module_test(torch.float32, torch.device('cuda'), [(1, 3)])
-@add_module_test(torch.float32, torch.device('cuda'), [(1, 3, 3)])
-@add_module_test(torch.float32, torch.device('cuda'), [(1, 3, 3, 3)])
+#@add_module_test(torch.float32, torch.device('cuda'), [(1, 3)])
+#@add_module_test(torch.float32, torch.device('cuda'), [(1, 3, 3)])
+@add_module_test(torch.float32, torch.device('cuda'), [(2, 3, 3, 3)],max_batch_size=3)
 def test_view_1d():
     return View(1, -1)
```

Comment:
It looks like the test case is hard-fixing the batch dimension to 1 here.  

In general, TensorRT engines are broadcast across the batch dimension, so operations that change the batch dimension aren't permitted.

Perhaps adding a test case with View(2, ...) would work for batch size to.  Or maybe even View(-1, ...) with other dimensions specified in full.